src/applications/common/schemaform/save-in-progress/SaveInProgressIntro.jsx

```diff
@@ -147,6 +163,19 @@ class SaveInProgressIntro extends React.Component {
         <br/>
       </div>
     );
+
+    if (this.props.downtime && !this.props.isLoggedIn) {
```

Comment:
I'm only blocking starting a form if you're not logged in, since a logged in user could fill in the form and save it, even if they can't submit.